unfortunately if_bridge.h from Linux 3.8+ is not self-contained and the struct
for ip6 is missing

http://bugs.gentoo.org/460262

this is not unheard of with Linux headers, for example, <linux/cdrom.h> needs
<limits.h> to get INT_MAX

--- libbridge/libbridge.h
+++ libbridge/libbridge.h
@@ -20,6 +20,7 @@
 #define _LIBBRIDGE_H
 
 #include <sys/socket.h>
+#include <netinet/ip6.h>
 #include <linux/if.h>
 #include <linux/if_bridge.h>
 
